src/site/content/en/blog/codelab-sign-in-form-best-practices/index.md

```diff
@@ -137,58 +137,54 @@ All of this happens because the `type="email"` attribute is applied to an `<inpu
        alt="The default email keyboard on iOS.">
 </figure>
 
-Try typing some text into the password input. The text is hidden by default because the
-`type="password"` attribute has been applied to the element.
+4. Type some text into the password input. 
+
+The text is hidden by default because the `type="password"` attribute has been applied to the element.
 
 * The `autocomplete`, `name`, `id`, and `type` attributes help browsers understand 
-the role of inputs in order to store data that can later be used for autofill. 
+the role of inputs in order to store data that can be used later for autofill. 
+
+5. Focus the email input on a desktop device and type some text. 
 
-Try focusing the email input on a desktop device and start typing.
-You can find the URL of your app by clicking **Fullscreen**
-![The Fullscreen icon](/images/glitch/fullscreen.svg). 
-If you've stored any email addresses in your browser, you'll probably see a popup that 
-allows you to select from those stored emails. This happens because the 
-`autocomplete="username"` attribute was applied to the email input.
+You can see the URL of your app when you click **Fullscreen** ![The Fullscreen icon](/images/glitch/fullscreen.svg. If you stored any email addresses in your browser, you probably see a dialog that lets you select from those stored emails. This happens because the `autocomplete="username"` attribute applied to the email input.
```

Comment:
```suggestion
You can see the URL of your app when you click **Fullscreen** ![The Fullscreen icon](/images/glitch/fullscreen.svg). If you stored any email addresses in your browser, you probably see a dialog that lets you select from those stored emails. This happens because the `autocomplete="username"` attribute applied to the email input.
```